Makefile

```diff
@@ -38,6 +38,11 @@ cover:
 	go test -coverprofile cover.out ./...
 	go tool cover -html=cover.out
 
+.PHONY: format
+format:
+	@echo goimports -d -w '**/*.go'
```

Comment:
Why the `@echo` ? If you want the command ont the next line to be printed, just remove the `@` from it, not need for an extra line for this

Reply:
Because it lists all files, which is just unreadable.

If I remove the `@` and the `@echo` then we get this:

```console
$ make format
goimports -d -w ./ast/ast.go ./ast/ast_test.go ./ast/expressions.go ./ast/literals.go ./ast/program.go ./ast/statements.go ./builtins/codecs.go ./builtins/builtins.go ./cmd/risor-api/main.go ./cmd/risor-api/main_test.go ./cmd/risor-docs/main.go ./cmd/risor-lsp/cache.go ./cmd/risor-lsp/completion.go ./cmd/risor-lsp/configuration.go ./cmd/risor-lsp/definition.go ./cmd/risor-lsp/execute.go ./cmd/risor-lsp/formatting.go ./cmd/risor-lsp/hover.go ./cmd/risor-lsp/main.go ./cmd/risor-lsp/server.go ./cmd/risor-lsp/stdio.go ./cmd/risor-lsp/symbols.go ./cmd/risor-lsp/unused.go ./cmd/risor/main.go ./cmd/risor/repl/repl.go ./cmd/risor/root.go ./cmd/risor-modgen/func.go ./cmd/risor-modgen/main.go ./compiler/code.go ./compiler/compiler.go ./compiler/compiler_test.go ./compiler/function.go ./compiler/instruction_iter.go ./compiler/store.go ./compiler/store_test.go ./compiler/symbol_table.go ./compiler/symbol_table_test.go ./compiler/symbols.go ./doctest/doctest_test.go ./errz/errz.go ./examples/go/quickstart/main.go ./examples/go/struct/main.go ./importer/importer.go ./internal/arg/args.go ./internal/arg/args_test.go ./internal/tmpl/string_template.go ./internal/tmpl/string_template_test.go ./lexer/lexer.go ./lexer/lexer_test.go ./limits/limits.go ./limits/standard_limits.go ./modules/all/all.go ./modules/aws/aws.go ./modules/aws/client.go ./modules/aws/config.go ./modules/aws/method.go ./modules/aws/stub.go ./modules/aws/util.go ./modules/base64/base64.go ./modules/base64/base64_test.go ./modules/bytes/bytes.go ./modules/dns/dns.go ./modules/exec/command.go ./modules/exec/exec.go ./modules/exec/result.go ./modules/filepath/filepath.go ./modules/filepath/filepath_test.go ./modules/fmt/fmt.go ./modules/http/client.go ./modules/http/fetch.go ./modules/http/fetch_test.go ./modules/http/http.go ./modules/http/request.go ./modules/http/response.go ./modules/image/image.go ./modules/image/image_object.go ./modules/jmespath/jmespath.go ./modules/json/json.go ./modules/kubernetes/stub.go ./modules/kubernetes/k8s.go ./modules/math/math.go ./modules/os/os.go ./modules/pgx/pgx.go ./modules/pgx/pgx_conn.go ./modules/rand/rand.go ./modules/regexp/regexp.go ./modules/regexp/regexp_object.go ./modules/regexp/regexp_test.go ./modules/sql/db.go ./modules/sql/sql.go ./modules/strconv/strconv.go ./modules/strings/strings.go ./modules/template/helpers.go ./modules/template/jsonpath.go ./modules/template/k8s.go ./modules/template/k8s_stub.go ./modules/template/module.go ./modules/template/render.go ./modules/template/render_k8s_test.go ./modules/template/render_test.go ./modules/template/template.go ./modules/time/time.go ./modules/uuid/uuid.go ./modules/vault/module.go ./modules/vault/stub.go ./modules/vault/vault.go ./modules/yaml/yaml.go ./modules/gha/gha.go ./object/base.go ./object/bool.go ./object/bool_test.go ./object/buffer.go ./object/builtin.go ./object/byte.go ./object/byte_slice.go ./object/cell.go ./object/color.go ./object/dir_entry.go ./object/dynamic_attr.go ./object/error.go ./object/errors.go ./object/file.go ./object/file_info.go ./object/file_iter.go ./object/file_mode.go ./object/file_test.go ./object/float.go ./object/float_slice.go ./object/float_test.go ./object/function.go ./object/go_field.go ./object/go_method.go ./object/go_method_test.go ./object/go_type.go ./object/int.go ./object/int_test.go ./object/iter_entry.go ./object/list.go ./object/list_iter.go ./object/list_test.go ./object/map.go ./object/map_iter.go ./object/module.go ./object/nil.go ./object/object.go ./object/object_test.go ./object/operations.go ./object/operations_test.go ./object/partial.go ./object/proxy.go ./object/proxy_test.go ./object/set.go ./object/set_iter.go ./object/slice_iter.go ./object/slice_iter_test.go ./object/sort.go ./object/string.go ./object/string_test.go ./object/time.go ./object/typeconv.go ./object/typeconv_test.go ./object/context_values.go ./op/op.go ./op/op_test.go ./os/localfs/localfs_test.go ./os/localfs/localfs.go ./os/nil_file.go ./os/os_test.go ./os/s3fs/s3_test.go ./os/s3fs/s3.go ./os/os.go ./os/simple.go ./os/virtual.go ./os/buffer_file.go ./os/in_memory_file.go ./os/in_memory_file_test.go ./parser/errors.go ./parser/parser.go ./parser/parser_test.go ./parser/precedence.go ./parser/test_helpers.go ./tests/benchmark/main.go ./tests/eval_test.go ./token/token.go ./token/token_test.go ./vm/code.go ./vm/run.go ./vm/frame.go ./vm/vm_test.go ./vm/vm.go ./cfg.go ./risor.go ./risor_test.go
```

Reply:
Sadly, goimports doens't understand `./...` or glob patterns. So you have to feed it each line individually.

So with the `@echo` part, we instead get this:

```console
$ make format
goimports -d -w **/*.go
```

Which is kind of the same, but more readable.

Reply:
What about using `gofumpt` ? It's a more strict `go fmt` and also organises imports.

Reply:
Yea sure! I've never used gofumpt before, but it does seem to have better built in support for traversing the file tree itself.

I tried running it and it has a lot of changes to the codebase:

```
 51 files changed, 31 insertions(+), 115 deletions(-)
```

Reply:
I ran gofumpt, but not on the generated files. Instead, I created a new PR to add gofumpt into the risor-modgen project directly: #179